src/common/api/types.ts

```diff
@@ -1723,7 +1723,7 @@ export interface OpenshiftVersion {
   /**
    * Level of support of the version.
    */
-  supportLevel: 'beta' | 'production' | 'custom';
+  supportLevel: 'beta' | 'production' | 'custom' | 'maintenance';
```

Comment:
is this coming from swagger? I don't see an updated swagger.yaml file here?

Reply:
this file should not be modified manually

Reply:
seems like `custom` is not part of this interface anymore
![image](https://user-images.githubusercontent.com/77008341/140045420-9ddfc1fc-aca1-4ae4-8135-fcba35bccd21.png)

Reply:
No, it's added manually. Should I update the swagger.yaml instead? @jkilzi

Reply:
yes please